packages/ai-bot/main.ts

```diff
@@ -100,43 +100,54 @@ async function sendStream(
     // If we've not got a current message to edit and we're processing text
     // rather than structured data, start a new message to update.
     if (message.type == ParsingMode.Text) {
-      // If there's no message to append to, just send the message
-      // If there's more than 20 pending messages, send the message
-      if (!append_to) {
-        let initialMessage = await sendMessage(
-          client,
-          room,
-          message.content,
-          append_to,
-        );
-        unsent = 0;
-        lastUnsentMessage = undefined;
-        append_to = initialMessage.event_id;
-      }
+      // remove general cruft
```

Comment:
Can you post some examples of messages that need cleanup? Would also be good if `cleanContent` had some tests

Reply:
Good call, added